zone/mob.h

```diff
@@ -208,6 +230,8 @@ class Mob : public Entity {
 	// Bot attack flag
 	Timer bot_attack_flag_timer;
 
+	std::vector<BotSpellSettings_Struct> _spellSettings;
```

Comment:
This is a very bot specific thing in `mob.h` without any notion of bot

`m_bot_spell_settings`